libraries-data-io/pom.xml

```diff
@@ -61,6 +61,52 @@
             <version>${org.assertj.core.version}</version>
             <scope>test</scope>
         </dependency>
+         <!-- gson api -->
```

Comment:
There's a mixture of indentation spaces and tabs. Let's apply spaces across-the-board to be like the rest of the file.